mars-rover-new/src/mars_rover_new/core.clj

```diff
@@ -48,7 +71,7 @@
   (let [str-seq (split-string plateau-size)
         count-test (= 2 (count str-seq))
         [x y] (map str->int str-seq)]
-    (and count-test x y (> x 0) (> y 0))))
+    (truthy (and count-test x y (> x 0) (> y 0)))))
```

Comment:
Please leave a comment explaining why you had to return a boolean value and why returning a `nil` wouldn't have worked.